src/pybind/mgr/ansible/tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+envlist = py27,py3
+skipsdist = true
+toxworkdir = {env:CEPH_BUILD_DIR}/ansible
+minversion = 2.8.1
```

Comment:
2.3

Reply:
I will change to see what happen ... no clues about why it was working before (o_o)